cmd/ui-errors.go

```diff
@@ -29,6 +29,12 @@ var (
 		"Browser can only accept `on` and `off` values. To disable web browser access, set this value to `off`",
 	)
 
+	uiErrInvalidWormValue = newUIErrFn(
+		"Invalid worm value",
```

Comment:
It might be better to rename worm to WORM since it is an acronym

Reply:
fixed